app/main.py

```diff
@@ -181,11 +188,15 @@ def move():
     # Route setter
     safety = safe(grid, me, data)
     route = path(me, foods[0], grid)
+    routeTail = path(me, tails[0], grid) # Modeled after Dennis' path finding for food, but replaces - hopefully - with tails.
 
     for item in safety:
         if item in route:
             output = item
             break
+        elif item in routeTail: # This way food is primary concern, tails secondary. Might consider changing this so that tails are primary until HP threshold reached, then foods primary.
```

Comment:
The main "thinking" behind the AI does not go here. It lies in the path function. =)

Reply:
Oh! It seems I have some more to learn with this syntax. Thanks again for the explanations and tips. I will take another look in the morning and make more sense of the AI. Cheers!